### PR TITLE
Update team with 2020-2021 board

### PR DESCRIPTION
**Pull request by JoshiNaman**
Updated members  page and mentor section

### CODE REVIEW

**Comment by shrijitsingh99** [comment]
Either you can be a mentor or member. For the remaining people, we can create a new section (#15). So this needs fixing.

Is the mentor list up to date, confirm once.

The page seems too long to, maybe a view more button? @Sarath18

**Comment by Sarath18** [comment]
Sounds like a good idea. Year wise team section will go hand in hand with this.

**Comment by shrijitsingh99** [comment]
Then for now let us just add the new team, and we can update the mentor list in a separate PR.
Don't think anyone will mind, for the time being, parallelly we develop #15,
@Sarath18 let discuss any mockups over there.

@JoshiNaman Please update only the team list and remove any senior members, don't add or remove any mentors for now.

**Comment by JoshiNaman** [comment]
add the new team as in only the 2nd years, right? and take out the third years from the mentor section for the time being

**Comment by shrijitsingh99** [comment]
> add the new team as in only the 2nd years, right? and take out the third years from the mentor section for the time being

Yup, remove current 3rd years from mentors list, as well as team. We can add them back after the year wise section is developed.

**Comment by aneesh404** [comment]
@JoshiNaman Parthesh Savla and Gaurav Thakur are in third year, remove them from team section.

**Comment by JoshiNaman** [comment]
i have made the changes to the branch, do I have to put the pull request again or it updates itself
I have pushed the changes

File: _data/mentors.yaml [hunk 1]
```diff
@@ -116,6 +116,7 @@
   linkedin: https://www.linkedin.com/in/tanuj-agrawal-88a149139
   github:
 
+
```
Comment (shrijitsingh99) [inline _data/mentors.yaml]: No need for this new empty line, remove it, please.

File: _site/index.html [hunk 2]
```diff
@@ -0,0 +1,594 @@
+<!DOCTYPE html>
```
Comment (shrijitsingh99) [inline _site/index.html]: Don't push the _site folder, it is auto-generated by Jekyll.

**Review by shrijitsingh99 — CHANGES_REQUESTED** [review]

**Comment by shrijitsingh99** [comment]
It auto-updates.
Just a tip for future, read [this](https://chris.beams.io/posts/git-commit/) on how to write good commit messages. 
Commit messages allows people later to understand what change was done in that commit, without going through the changed code.

For this time will just squash the commits, since its a small change.

**Comment by shrijitsingh99** [comment]
Omkar also needs to be removed, Akhil is Automation Head, that needs to be added.

**Review by shrijitsingh99 — APPROVED** [review]

**Pull request merged**